backend/src/personnel/personnel.service.ts

```
@@ -149,6 +171,25 @@ export class PersonnelService {
     }
   }
 
+  async createPerson(personnel: CreatePersonnelDTO): Promise<PersonnelEntity> {
+    try {
+      return await this.personnelRepository.save(
+        this.personnelRepository.create(new PersonnelEntity(personnel)),
+      );
+    } catch (e) {
+      console.log(e);
```

Should probably throw an error